test/jdk/java/lang/String/TranslateEscapes.java

```diff
@@ -94,6 +110,22 @@ static void verifyEscape(String string, char ch) {
         }
     }
 
+    static void verifyEscape(String string1, String string2) {
+        if (!string1.translateEscapes().equals(string2)) {
+            System.err.format("\"%s\" not escaped \"%s\"%n", string1, string2);
+            throw new RuntimeException();
+        }
+    }
+
+    static void verifyUnicodeEscape(String string) {
+        try {
+            string.translateEscapes();
+            System.err.format("\"%s\" should be an error%n", string);
+            throw new RuntimeException();
+        } catch (IllegalArgumentException ex) {
+        }
+    }
```

Comment:
The method name implies valid unicode escape sequences, but they are all invalid.
The method name could be "verifyIllegalUnicodeEscape`.

Reply:
Changed